src/proto/srv/srv.proto

```diff
@@ -0,0 +1,34 @@
+//
+// (C) Copyright 2019 Intel Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+// The Government's rights to use, modify, reproduce, release, perform, display,
+// or disclose this software are subject to the terms of the Apache License as
+// provided in Contract No. 8F-30005.
+// Any reproduction of computer software, computer software documentation, or
+// portions thereof marked with this legend must also reproduce the markings.
+//
+
+// This file defines the messages used by DRPC_MODULE_SRV.
+
+syntax = "proto3";
+package srv;
+
+message NotifyReadyReq {
+	string uri = 1;
+	uint32 nctxs = 2;
+}
+
+// message NotifyReadyResp == nil
```

Comment:
Commented out line

Reply:
Not quite; this is a comment indicating that the response is always nil.

Reply:
I've changed it to be a sentence instead of something looks like code.